kaldb/src/main/java/com/slack/kaldb/metadata/core/MetadataSerializer.java

```diff
@@ -23,6 +23,10 @@ default ModelSerializer<T> toModelSerializer() {
     return new ModelSerializer<>() {
       @Override
       public byte[] serialize(T model) {
```

Comment:
not related to this change, but perhaps the variable `model` is confusing? Especially now that we have modelClient? Would `metadata` be a better name here?